src/fprime_gds/common/fpy/bytecode/directives.py

```diff
@@ -11,29 +11,34 @@
 except ImportError:
     UNION_TYPES = (typing.Union,)
 
-from dataclasses import dataclass, fields, astuple
+from dataclasses import dataclass, fields
 from typing import ClassVar
 import typing
 from typing import Union
-from pathlib import Path
 import struct
-import zlib
-from fprime.common.models.serialize.type_base import BaseType
+from fprime.common.models.serialize.type_base import BaseType as FppValue
 from fprime.common.models.serialize.numerical_types import (
-    U32Type,
-    U16Type,
-    U64Type,
-    U8Type,
-    I64Type,
-    I16Type,
-    I32Type,
-    I8Type,
-    F32Type,
-    F64Type,
+    U8Type as U8Value,
```

Comment:
I don't love these "as" statements as they change the name of types and thus obfuscate what they are.